packages/core/src/wallet/IndyWallet.ts

```diff
@@ -33,6 +34,14 @@ export class IndyWallet implements Wallet {
   public constructor(agentConfig: AgentConfig) {
     this.logger = agentConfig.logger
     this.indy = agentConfig.agentDependencies.indy
+
+    const { walletConfig, fileSystem } = agentConfig
+
+    if (walletConfig?.storageType === WalletStorageType.Postgres) {
+      if (walletConfig.storageConfig && walletConfig.storageCreds && fileSystem?.loadPostgresPlugin) {
+        fileSystem?.loadPostgresPlugin(walletConfig.storageConfig, walletConfig.storageCreds)
+      }
```

Comment:
If the wallet storage is set to postgres, I think we should throw an error if the required parameters are not set. I think we should also throw an error if `loadPostgresPlugin` is not defined and wallet storage type is postgres.